src/Command/BaseCommand.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace DepDoc\Command;
+
+use DepDoc\PackageManager\ComposerPackageManager;
+use DepDoc\PackageManager\NodePackageManager;
+use DepDoc\Parser\AbstractParser;
+use DepDoc\Parser\MarkdownParser;
+use DepDoc\Validator\PackageValidator;
+use DepDoc\Writer\AbstractWriter;
+use DepDoc\Writer\MarkdownWriter;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+abstract class BaseCommand extends Command
+{
+    /** @var ComposerPackageManager */
+    protected $managerComposer;
+    /** @var NodePackageManager */
+    protected $managerNode;
+    /** @var AbstractParser */
+    protected $parser;
+    /** @var AbstractWriter */
+    protected $writer;
+    /** @var PackageValidator */
+    protected $validator;
+    /** @var SymfonyStyle */
+    protected $io;
+
+    public function __construct(string $name = null)
+    {
+        parent::__construct($name);
+
+        $this->managerComposer = new ComposerPackageManager();
+        $this->managerNode = new NodePackageManager();
+        $this->parser = new MarkdownParser();
+        $this->writer = new MarkdownWriter();
+        $this->validator = new PackageValidator();
```

Comment:
Maybe we should consider only instantiating those dependencies needed for that command, e.g. UpdateCommand doesn't need Validator, but then again this might be the only thing so it might just be easier this way!